firebase-vscode/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Change Log
```

Comment:
I made this change to automate our changelog - we can put change log entries for the next release under NEXT, and the script will automatically replace it with the next version when its run.